.github/workflows/e2e.yaml

```
@@ -1,11 +1,15 @@
 name: e2e
 on:
   pull_request:
-    paths-ignore:
-      - "**.md"
-      - "CODEOWNERS"
-      - ".gitignore"
-      - "docs/**"
+    paths:
+      - "*"
+      - "!.github/**"
+      - ".github/workflows/e2e.yml"
+      - "!**.md"
+      - "!CODEOWNERS"
+      - "!.gitignore"
+      - "!docs/**"
+      - "!Makefile"
```

This feels a little complicated now, would this work instead?

```suggestion
    paths-ignore:
      - ".github/**"
      - "!.github/workflows/e2e.yaml"
      - "**.md"
      - "CODEOWNERS"
      - ".gitignore"
      - "docs/**"
      - "Makefile"
```

---

I don't believe paths-ignore supports "!"

---

[Looks like you're right.](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore)